### DISCUSSION TITLE
Can't display multiple <Coordinates/>

### DISCUSSION THREAD

**Discussion by Nikhil22** [RodrigoHamuy/react-three-map#114] [General]
Inside a Canvas I am rendering a 3D model for every element in `reservedHomeLocations`, like this:

```ts
 <Canvas longitude={startingLngLat.lng} latitude={startingLngLat.lat} overlay>
        {reservedHomeLocations?.map((l, idx) => {
          const modelPath = getHomeAttributesObj(l.homeTypeId)?.modelPath;
          return (
            <Coordinates
              key={idx}
              latitude={l.coordinates[1]}
              longitude={l.coordinates[0]}
            >
              <Suspense fallback={<></>}>
                <object3D>
                  {modelPath && <HomeModel3D modelPath={modelPath} />}
                </object3D>
              </Suspense>
            </Coordinates>
          );
        })}
<Canvas/>
```

The problem is, despite `reservedHomeLocations` being multiple elements long, only one 3D model (the last one) shows up. The model paths are the same, the coordinates are all different.

No errors in the console.

**Comment by RodrigoHamuy** [comment]
@Nikhil22 , you have a sandbox to test your issue? Multi coordinates are working fine here https://rodrigohamuy.github.io/react-three-map/?story=multi-coordinates--default
